simtools/data_model/model_data_writer.py

```diff
@@ -84,7 +83,7 @@ def write(self, metadata=None, product_data=None):
         if product_data is not None:
             self.write_data(product_data=product_data)
 
-    def validate_and_transform(self, metadata=None, product_data=None, validate_schema_file=None):
```

Comment:
we should remove metadata from the docstring as well

Reply:
Yes, done.